ui/css/design-system/colors.scss

```diff
@@ -118,7 +118,7 @@ $color-map: (
   'ui-4': $ui-4,
   'white': $ui-white,
   'black': $ui-black,
-  'grey': $ui-grey,
+  'gray': $neutral-grey,
```

Comment:
This was recently changed to `grey` for spelling consistency, any reason for the change back?

Reply:
No reasons, inconsistency in codebase confused me, since `COLORS.GRAY` was. there when I wrote this code.

Let me fix it.

Reply:
Gotcha, thank you! :D